language-support/ts/defs.bzl

```diff
@@ -1,21 +1,24 @@
 # Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-load("@language_support_ts_deps//typescript:index.bzl", "tsc")
+load("@build_bazel_rules_nodejs//:providers.bzl", "DeclarationInfo", "LinkablePackageInfo")
+load("@npm_bazel_typescript//:index.bzl", "ts_project")
 
 def _da_ts_library_impl(ctx):
+    path = "/".join([p for p in [ctx.bin_dir.path, ctx.label.workspace_root, ctx.label.package] if p])
```

Comment:
Use `paths.join` from `bazel_skylib`, note, it already drops empty components..

Reply:
Thank, I had just copied that from `rules_nodejs`.